kubernetes/deployment-production.tmpl

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: caesar-production-app
+  labels:
+    app: caesar-production-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: caesar-production-app
+  template:
+    metadata:
+      labels:
+        app: caesar-production-app
+    spec:
+      containers:
+        - name: caesar-production-app
+          image: zooniverse/caesar:__IMAGE_TAG__
+          env:
+            - name: RAILS_ENV
+              value: production
+            - name: DOTENV_FILE
+              value: /run/secrets/environment
+            - name: PORT
+              value: "81"
+          volumeMounts:
+            - name: caesar-production-environment
+              mountPath: "/run/secrets/environment"
+              subPath: "environment"
+              readOnly: true
+            - name: static-assets
+              mountPath: "/static-assets"
+          lifecycle:
+            postStart:
+              exec:
+                command: ["/bin/bash", "-c", "cp -R /app/public/* /static-assets"]
+        - name: caesar-production-nginx
+          image: zooniverse/apps-nginx:xenial
+          ports:
+            - containerPort: 80
+          volumeMounts:
+            - name: static-assets
+              mountPath: "/static-assets"
+            - name: caesar-nginx-config
+              mountPath: "/etc/nginx-sites"
+      volumes:
+        - name: caesar-production-environment
+          secret:
+            secretName: caesar-production-environment
+        - name: static-assets
+          emptyDir: {}
+        - name: caesar-nginx-config
+          configMap:
+            name: caesar-nginx-conf
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: caesar-production-sidekiq
+  labels:
+    app: caesar-production-sidekiq
+spec:
+  replicas: 1
```

Comment:
I'd say out of scope for this PR / initial deployment setup but it looks like we can add the ability to autoscale sidekiq based on say cpu metrics (and others) 

https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics

Reply:
Agreed and extremely worth pursuing. We had a lot of flexibility to scale with straight AMIs and that _should_ be even more seamless in k8s so long as those metrics are readily available.

Reply:
`kubectl top` works so we must have the metrics-server installed in the cluster, but I don't see its pod running. Chances are it's as easy as that one `kubectl autoscale ...` line. I can test this whole thing on a staging deployment, with "CPU > 75" or so as a metric to start with.